tests/test_helper.js

```diff
@@ -132,9 +130,7 @@ function assertValidHTML(response, done) {
     });
 }
 
-function preFetch(uri, cb, http) {
-    http = typeof http === 'undefined' ? require('http') : http;
-
+function preFetch(uri, cb, http = require('http')) {
```

Comment:
And I remember seeing that you can set defaults in args, but I forgot during implementation, nice!